### ISSUE TITLE
Table Name

### ISSUE THREAD

**Issue by zivko** [skttl/Our.Umbraco.FullTextSearch#5]
CreateCacheTable.cs - line 20, think this need to be change from

```
if (TableExists("BlogComments") == false)

to

if (TableExists("FullTextCache") == false)
```

**Comment by skttl** [comment]
I think you are right, maybe you want to do a PR?

**Comment by stefankip** [comment]
Guess not :-p

**Comment by skttl** [comment]
Fixed in 506c1693f8f1386873c87489c314afce17470056
